lib/models.rb

```diff
@@ -10,6 +10,10 @@ def initialize(slug:, number:, name:)
     @number = number
     @name = name
   end
+
+  def dir_name
```

Comment:
```suggestion
  def solution_dir
```

(similar to `Course#solutions_dir` and `Course#dir`)